src/mgr/MgrStandby.cc

```diff
@@ -114,6 +140,18 @@ void MgrStandby::handle_conf_change(
   }
 }
 
+int MgrStandby::asok_command(std::string_view cmd, const cmdmap_t& cmdmap, Formatter* f, std::ostream& errss)
+{
+  dout(10) << __func__ << ": " << cmd << dendl;
+  if (cmd == "status") {
+    f->open_object_section("status");
+    f->close_section();
+    return 0;
+  } else {
+    return -ENOSYS;
```

Comment:
Nit:

`ENOSYS` is for "function not supported". Does a command other than "status" qualify as a function? Why not a more general `ENOTSUP`?

Reply:
`ENOSYS` conventionally indicates that a requested method/function/syscall is not known to the kernel (or in this case, the ceph daemon). That would be appropriate in this context.

`ENOTSUP` is more for the case where the system understands the request but cannot implement it because of some contextual problem.

Reply:
ACK, thanks for clarifying!